kustomizations/apps/data-hub-configs/kubernetes-pipeline--prod.config.yaml

```diff
@@ -7,9 +7,9 @@ kubernetesPipelines:
         max_active_runs: 1
         tags:
           - 'Kubernetes'
-    image: 'docker.io/elifesciences/data-hub-core-dags:latest'
+    image: '${data_hub_core_dags_stable_image_repo:-docker.io/elifesciences/data-hub-core-dags}:${data_hub_core_dags_stable_image_tag:-latest}'
```

Comment:
In our main deployment we have something like this:

```yaml
      image:
        repository: docker.io/elifesciences/data-hub-with-dags_unstable # {"$imagepolicy": "data-hub:data-hub-unstable:name"}
        tag: develop-8ac00e8c-20241023.1644 # {"$imagepolicy": "data-hub:data-hub-unstable:tag"}
```

i.e. the image tag is actually committed. Does it work differently here because it is a config map?

Reply:
Actually the actual values are still committed to where the placeholders are defined.

Reply:
Another question is, do we need the default values?